src/plugins/index.js

```diff
@@ -106,6 +107,12 @@ export function getActivityTrackingPlugin({ agent, isProd }) {
 	};
 }
 
+export function getClickTrackingPlugin() {
```

Comment:
also, having the 'getters' makes it slightly easier to mix-and-match plugins for integration testing.